src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/IPAddressUtil.cs

```diff
@@ -34,7 +34,7 @@ public static bool IsMulticast(IPAddress address)
         public static unsafe IPAddress GetIPAddressFromNativeInfo(Interop.Sys.IpAddressInfo* addressInfo)
         {
             byte[] ipBytes = new byte[addressInfo->NumAddressBytes];
-            fixed (byte* ipArrayPtr = ipBytes)
+            fixed (byte* ipArrayPtr = &ipBytes[0])
```

Comment:
How do we know that NumAddressBytes isn't 0?  If it could be, this will now throw an index out of range exception, whereas before this we would instead get an argument exception from the IPAddress ctor below.